src/RipplePage.js

```diff
@@ -0,0 +1,91 @@
+import React, {Component} from 'react';
+import ComponentPage from './ComponentPage.js';
+import HeaderBar from './HeaderBar.js';
+import {MDCRipple} from '@material/ripple';
+
+import './styles/RipplePage.scss';
+
+const RipplePage = () => {
+  return (
+    <div>
+      <HeaderBar />
+      <ComponentPage
+        hero={<RippleHero />}
+        title='Ripple'
+        description=''
+        designLink='https://material.io/guidelines/motion/choreography.html#choreography-radial-reaction'
+        docsLink='https://material.io/components/web/catalog/ripples/'
+        sourceLink='https://github.com/material-components/material-components-web/tree/master/packages/mdc-ripple'
+        demos={<RippleDemos />}
+      />
+    </div>
+  );
+}
+
+class RippleHero extends Component {
```

Comment:
the hero ripple needs a bounding box, its not obvious where the ripple is right now.

Reply:
It has a bounding box. It's the entire middle portion of the hero section.